index.d.ts

```diff
@@ -24,7 +24,7 @@ declare module "p-iteration" {
    */
   export const forEachSeries: <T>(
     array: T[],
-    callback: (currentValue: T, index: number, array: T[]) => void,
+    callback: (currentValue: T, index: number, array: T[]) => Promise<void>,
     thisArg?: any
   ) => Promise<void>;
```

Comment:
What about specifying void here (and line 16) as the returned type (i.e. undefined)? Does that bring everything in line and get around the eslint warning? @toniov